airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -43,6 +44,9 @@ definitions:
       state:
         description: "schema message: the state. Must be the last message produced. The platform uses this information"
         "$ref": "#/definitions/AirbyteStateMessage"
+      trace:
+        description: "trace message: a message to provide human readable information to users"
```

Comment:
It is important to not call it human readable. It is really about providing structured information about the connector.

Reply:
Perhaps alternative language might be: 
> "trace message: a structured message to share information about connector and sync performance with external consumers"